server/config.ts

```diff
@@ -58,23 +58,23 @@ export default {
       apiClientSecret: get('API_CLIENT_SECRET', 'clientsecret', requiredInProduction),
     },
     allocationsService: {
-      url: get('ALLOCATIONS_SERVICE_URL', 'http://localhost:8091', requiredInProduction),
+      url: get('ALLOCATIONS_SERVICE_URL', 'http://localhost:9091', requiredInProduction),
```

Comment:
with this change can we remove https://github.com/ministryofjustice/manage-a-workforce-ui/blob/main/feature.env#L9...L11?